stellarphot/gui_tools/seeing_profile_functions.py

```diff
@@ -529,6 +237,8 @@ def load_fits(self, file):
             Filename to open.
         """
         self.fits_file.load_in_image_widget(self.iw)
+        self.object_name = self.fits_file.object
+        self.exposure = self.fits_file.header["EXPOSURE"]
```

Comment:
There are several keywords that get used for EXPOSURE, in fact in lines 34-35 of photometry.py we defined

```
# Allowed FITS header keywords for exposure values
EXPOSURE_KEYWORDS = ["EXPOSURE", "EXPTIME", "TELAPSE", "ELAPTIME", "ONTIME", "LIVETIME"]
```

You might want to loop through these choices and figure out which is used in the FITS file...

Reply:
Thanks for the suggestion, done!